easybuild/easyconfigs/o/openCARP/openCARP-17.0-foss-2023b.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'openCARP'
+version = '17.0'
+
+homepage = 'https://opencarp.org'
+description = "openCARP is an open cardiac electrophysiology simulator for in-silico experiments."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['https://git.opencarp.org/openCARP/openCARP/-/archive/v%(version)s/']
+sources = ['openCARP-v%(version)s.tar.gz']
+checksums = [
+    '6439b6b8e6a265cd6cfe39f59507418739d936de6f6e059b2d0f468df816d26e',  # openCARP-v17.0.tar.gz
+]
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('gengetopt', '2.23'),
+    ('pkg-config', '0.29.2'),
```

`pkg-config` is no longer maintained, `pkgconf` is (mostly) a drop-in replacement for it, can we try using that instead?

```suggestion
    ('pkgconf', '2.0.3'),
```

(and then remove the `pkg-config` easyconfig file you're added, see https://docs.easybuild.io/integration-with-github/?h=github#github_new_update_pr_delete